libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts

```diff
@@ -4,55 +4,55 @@ import { environment } from '../environments/environment'
 
 type StringOrNull = string | null
 
+// Fjölskyldumál
+const FJOLSKYLDUMAL = '4vQ4htPOAZvzcXBcjx06SH'
+
+// Skírteini
+const SKIRTEINI = '7nWhQCER920RakQ7BZpEmV'
+
+// Andlát og dánarbú
+const ANDLAT_OG_DANARBU = '2TkJynZlamqTHdjUziXDG0'
+
+// Þinglýsingar, staðfestingar og skráningar
+const THINGLYSINGAR = '6K9stHLAB2mEyGqtqjnXxf'
+
+// Gjöld og innheimta
+const GJOLD_OG_INNHEIMTA = '5u2M09Kw3p1Spva1GSuAzB'
+
+// Löggildingar
+const LOGGILDINGAR = 'WrQIftmx61sHJMoIr1QRW'
+
+// Vottorð
+const VOTTORD = '76Expbwtudon1Gz5lrKOit'
+
+// Lögráðamál
+const LOGRADAMAL = '4tvRkPgKP3kerbyRJDvaWF'
+
+// Önnur þjónusta sýslumanna
+const ONNUR_THJONUSTA_SYSLUMANNA = '4LNbNB3GvH3RcoIGpuZKhG'
+
+// Leyfi
+const LEYFI = '7HbSNTUHJReJ2GPeT1ni1C'
+
+// Fullnustugerðir
+const FULLNUSTUGERDIR = '7LkzuYSzqwM7k8fJyeRbm6'
+
+// Default/fallback email address
+const DEFAULT = 'default'
+
 type Categories =
-  /**
-   * Fjölskyldumál
-   */
-  | '4vQ4htPOAZvzcXBcjx06SH'
-  /**
-   * Skírteini
-   */
-  | '7nWhQCER920RakQ7BZpEmV'
-  /**
-   * Andlát og dánarbú
-   */
-  | '2TkJynZlamqTHdjUziXDG0'
-  /**
-   * Þinglýsingar, staðfestingar og skráningar
-   */
-  | '6K9stHLAB2mEyGqtqjnXxf'
-  /**
-   * Gjöld og innheimta
-   */
-  | '5u2M09Kw3p1Spva1GSuAzB'
-  /**
-   * Löggildingar
-   */
-  | 'WrQIftmx61sHJMoIr1QRW'
-  /**
-   * Vottorð
-   */
-  | '76Expbwtudon1Gz5lrKOit'
-  /**
-   * Lögráðamál
-   */
-  | '4tvRkPgKP3kerbyRJDvaWF'
-  /**
-   * Önnur þjónusta sýslumanna
-   */
-  | '4LNbNB3GvH3RcoIGpuZKhG'
-  /**
-   * Leyfi
-   */
-  | '7HbSNTUHJReJ2GPeT1ni1C'
-  /**
-   * Fullnustugerðir
-   */
-  | '7LkzuYSzqwM7k8fJyeRbm6'
-  /**
-   * Default/fallback email address
-   */
-  | 'default'
+  | typeof FJOLSKYLDUMAL
+  | typeof SKIRTEINI
+  | typeof ANDLAT_OG_DANARBU
+  | typeof THINGLYSINGAR
+  | typeof GJOLD_OG_INNHEIMTA
+  | typeof LOGGILDINGAR
+  | typeof VOTTORD
+  | typeof LOGRADAMAL
+  | typeof ONNUR_THJONUSTA_SYSLUMANNA
+  | typeof LEYFI
+  | typeof FULLNUSTUGERDIR
+  | typeof DEFAULT
```

Comment:
```suggestion
  | string
```
This doesn't really make sense to me. The types are all `string`.

Reply:
Aren't the types the actual possible value, like typeof FJOLSKYLDUMAL is the id, right?